nsv13/code/__HELPERS/names.dm

```diff
@@ -38,3 +38,14 @@ var/list/ship_name_suffix = list("slayer","blazer","bringer","slapper","twirler"
 	var/name = ""
 	name = pick(ship_name_prefix) + pick(ship_name_suffix)
 	return capitalize(name)
+
+/proc/generate_fighter_name()
+	var/name = ""
+	var/randomizer = rand(0,1)
+	var/chooser = FALSE
+	if(randomizer == 0)
+		chooser = FALSE
+	else
+		chooser = TRUE
+	name = pick(GLOB.fighter_noun) + " of " + "[chooser ? pick(GLOB.fighter_moon) : pick(GLOB.fighter_planet)]"
+	return capitalize(name)
```

Comment:
```suggestion
	name = pick(GLOB.fighter_noun) + " of " + "[pick(0,1) ? pick(GLOB.fighter_moon) : pick(GLOB.fighter_planet)]"

```
Just put the randomization in here and it'll work perfectly fine, no need to overcomplicate things.
Your text is all already capitalized in the file, also don't forget to put empty lines at the end of your files